### DISCUSSION TITLE
align en rojo y stile no funciona?

### DISCUSSION THREAD

**Discussion by venegas55** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#199] [Q&A] [answered]
hehe hola, estoy haciendo la práctica de la pastelería y quería meter un poco de CSS (por intentar adelantarme), y me encuentro con 2 cosas....

primero que al hacer tablas o divs si les pongo el atributo align me sale en rojo... (aunque si haga la alineación)

![image](https://user-images.githubusercontent.com/89881553/156297898-20ded0ca-9b28-4264-8d07-625ddf69575f.png)

además, generé una clase (.paddingBetwenCols td th) con la que quiero dar un poco de espacio entre las columnas y no hace nada pese a que le puse la clase a la tabla. (puse td y th después de la clase para que haga la restricción en la tabla y en headers y datos de la tabla)...

saludos!

**Comment by Ingrzs** [comment]
Creo es porque el align es para que lo uses en css y lo estas usando para html, y te esta diciendo que no lo estas usando de la manera "profesiónal" podria decirse (como lo dice w3c)
Si vas a centrar los td hazlo en css

**Comment by HctrGC** [comment]
Para que tus estilos puedan ser aplicados necesitas utilizarlo de la siguiente manera:  style="text-align: center;", style="align-content: center;" o style="align-items: center;", dependiendo a que etiqueta se la apliques. Prueba con las tres y ve que comportamiento toma tu texto.

Pda. Editado porque olvidé el punto y coma.

**Comment by hackit92** [comment]
Hola! Como ya te mencionaron con anterioridad es un error de sintaxis al mezclar html con css, a modo de buena práctica te recomiendo tener aparte tu hoja de estilos y ahi asignar las clases o ids según requieras para mantener tu codigo limpio y legible, de igual manera los comentarios son de mucha ayuda, éxito!

**Comment by Josue-HernandezL** [comment]
ponle directamente a la etiqueta <table> ub style="text-align: center;" ese es tu error, no lo estas agregando completo

**Comment by JuanSalvador73** [comment]
Como sugieren arriba, esa propiedad va en los estilos, no en las etiquetas. Y aunque puedes poner estilos directamente en el HTML, no es lo mas aconsejable, sino que los CSS sean externos.

**Comment by AngelCruzO** [comment, marked as answer]
Tu atributo align si existe solo que sirve para centrar texto directo en el HTML, para poder centrar div,section,header,main u otras etiquetas HTML necesitas el CSS. Te dejo un link donde te muestra la referencia de atributos de HTML, donde existen algunos que se utilizan para estilos directos sin tener que utilizar CSS.

Pero ya no son muy utilizados, [Referencia de atributos HTML](https://developer.mozilla.org/es/docs/Web/HTML/Attributes)

**Comment by venegas55** [comment]
Muchas gracias, voy a intentar meter esto en mi CSS!

salu2!

**Reply by AngelCruzO** [reply]
Las referencias no es para CSS solo para etiquetas HTML, para centrar divs ocupa las respuestas que ya te han dado antes
